src/NumericalAlgorithms/LinearOperators/Mass.cpp

```diff
@@ -0,0 +1,48 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "NumericalAlgorithms/LinearOperators/Mass.hpp"
+
+#include <ostream>
```

Comment:
The code guidelines say to #include Parallel/Printf.hpp, which is safe to use in parallel, instead of <iostream>.